packages/steppp/src/index.ts

```diff
@@ -55,24 +55,38 @@ function Steppp(element: HTMLElement, options: any = defaultOptions): Instance {
     return buildAnimation(args);
   };
 
+  const getFrames = (
```

Comment:
This method needs to exist so I can get the calculated frames before I want to trigger the animation.